File: RentalManager.Infrastructure/Commands/UpdateRentalAgreement.cs
using System.ComponentModel.DataAnnotations;

namespace RentalManager.Infrastructure.Commands
{
    public class UpdateRentalAgreement
    {
        public int EmployeeId { get; set; }
        public bool IsActive { get; set; }
        public int ClientId { get; set; }
        public List<int> RentalEquipmentIds { get; set; }
        public string? Comment { get; set; }
        public int Deposit { get; set; }
        public int TransportFrom { get; set; }
        public int? TransportTo { get; set; }
        public DateTime DateAdded { get; set; }
    }
}

File: RentalManager.WebApp/src/references/codegen/API/models/UpdateRentalAgreement.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type UpdateRentalAgreement = {
    employeeId?: number;
    isActive?: boolean;
    clientId?: number;
    rentalEquipmentIds?: Array<number> | null;
    comment?: string | null;
    deposit?: number;
    transportFrom?: number | null;
    transportTo?: number;
    dateAdded?: string;
};
